worlds/stardew_valley/data/fish_data.py

```diff
@@ -11,6 +11,7 @@ class FishItem(GameItem):
     locations: Tuple[str]
     seasons: Tuple[str]
     difficulty: int
+    mod_name: str
 
     def __repr__(self):
```

Comment:
For the time being, this inclusion merely sets up the data structure for mod data.  Current mods don't add any new modded fish, but there will be modded fish (Stardew Valley Expanded adds several, maybe around twenty).  So currently there's no "string representation of the mod name for the fish" because its still just Stardew Valley right now, so I couldn't add a string so I'm not sure what to do with this.

Reply:
I meant the method __repr__(). So that when printing the FishItem, whether in a debugger or else, it also includes the new mod_name string